hazelcast-jet-core/src/test/java/com/hazelcast/jet/MulticastDiscoveryTest.java

```diff
@@ -37,7 +38,7 @@
 import static org.junit.Assert.assertEquals;
 
 @RunWith(HazelcastSerialClassRunner.class)
-@Category(SerialTest.class)
+@Category({SerialTest.class, SlowTest.class})
```

Comment:
`SerialTest` can be removed here. All slow tests are running in serial mode.

Reply:
I think we should keep it, since it means different things, what do you think?

Reply:
Ok, we can keep it - currently there is no difference since `serial-test` is skipped in `nightly` profile and all tests are executed in `regular` execution (which executes them serially for `nightly` profile). However maybe in the future we can split `nightly` profile also to run in two part (`regular` and `serial`) - then it can be useful.